pkg/scheduler/metrics/metrics.go

```diff
@@ -87,6 +87,30 @@ var (
 			Name:      "total_preemption_attempts",
 			Help:      "Total preemption attempts in the cluster till now",
 		})
+	AssumeBindVolumesLatency = prometheus.NewHistogram(
```

Comment:
I know the function name is assume and bind, but I think it makes more sense for the metric to be named just "AssumeVolumesLatency" because the function doesn't actually do the bind operation, it just queues it.

Reply:
Done